docs/networking.md

```diff
@@ -33,3 +33,30 @@ To disable it, start each server with the `disable: rke2-ingress-nginx` option i
 # Nodes Without a Hostname
 
 Some cloud providers, such as Linode, will create machines with "localhost" as the hostname and others may not have a hostname set at all. This can cause problems with domain name resolution. You can run RKE2 with the `node-name` parameter and this will pass the node name to resolve this issue.
+
+# CNI options
+
+### Replacing the default CNI
+
+By default RKE2 deploys the Canal CNI which uses flannel for networking and provides calico-felix to enable networkpolicy support.  If you wish to use a CNI other than Canal, you can disable it by adding the `disable: rke2-canal` option in your configuration file.
+
+### Running Calico as your CNI
+
+In order to deploy Calico as your CNI you will need to set a few options in your RKE2 config file.
+
+```
+service-cidr: "10.41.0.0/16"
+cluster-cidr: "10.42.0.0/16"
```

Comment:
Does this need to be set to the default calico cidr range: `192.168.0.0/16`? Or the calico cidr range needs to be modified to the cluster cidr.

Reply:
@joshrwolf and @cnuber if you two can resolve this comment one way or the other, we can merge.